src/modules/sharing/sharing.controller.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { SharingController } from './sharing.controller';
+import { SharingService } from './sharing.service';
+import { UuidDto } from '../../common/uuid.dto';
+import { createMock } from '@golevelup/ts-jest';
+import { Chance } from 'chance';
+
+describe('SharingController', () => {
+  let controller: SharingController;
+  let sharingService: SharingService;
+
+  beforeEach(async () => {
+    sharingService = createMock<SharingService>();
+    controller = new SharingController(sharingService);
+  });
+
+  describe('get public sharing folder size', () => {
+    it('When request the get sharing size method, then it works', async () => {
+      const sharingId = new Chance().guid();
```

Comment:
Hey @edisonjpadilla, let's use the fixtures, therefore you have the centralization in that generator (`newSharing`). If we change the format of the id or whatever, you will need to change all the references in the code.